### PR TITLE
created banner

### PR DESCRIPTION
**Pull request by Deon-basson**

### CODE REVIEW

File: src/Components/banner/banner.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
+.App {
+    display: flex;
```
Comment (mdsoftwarelab) [inline src/Components/banner/banner.css]: There are built-in css classes for this. Look at layout.css

File: src/Components/banner/banner.css [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    justify-content: center;
+    align-items: center;
+    margin: 0;
+    padding-top: 1em;
+  }
+  
+  .slider-container {
+    display: grid;
```
Comment (mdsoftwarelab) [inline src/Components/banner/banner.css:8]: this too

File: src/Components/banner/banner.css [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  }
+  
+  .slider-image {
+    width: 100%;
+    height: auto;
+    max-height: 500px; 
+    object-fit: cover;
+  }
+  
+  button {
```
Comment (mdsoftwarelab) [inline src/Components/banner/banner.css]: move button logic to buttons.css in the theme folder

File: src/Components/banner/banner.css [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    border-radius: 50%;
+    z-index: 1; 
+  }
+  
+
```
Comment (mdsoftwarelab) [inline src/Components/banner/banner.css]: remove double white spaces

File: src/Components/banner/banner.jsx [hunk 5]
```diff
@@ -0,0 +1,51 @@
+import React, {useState, useEffect } from "react";
+import './banner.css'
+
+
```
Comment (mdsoftwarelab) [inline src/Components/banner/banner.jsx]: remove whitespace

File: src/Components/banner/banner.jsx [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+    const images = [
+      "/images/bannerImage.jpg",
+      "/images/banner2.png"
+    ]
+
```
Comment (mdsoftwarelab) [inline src/Components/banner/banner.jsx:8]: remove whitespace

File: src/Components/banner/banner.jsx [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+    function Banner() {
+      const [currentIndex, setCurrentIndex] = useState(0);
+    
+      const moveSlide = (direction) => {
+        if (direction === 'next') {
+          setCurrentIndex((prevIndex) => (prevIndex + 1) % images
+          .length);
+        } else if (direction === 'prev') {
```
Comment (mdsoftwarelab) [inline src/Components/banner/banner.jsx]: Do we need a "else if" here? would there be another case except for "next" and "prev"?

File: src/Components/banner/banner.css (hunk 1 above)
Comment (Deon-basson) [reply src/Components/banner/banner.css]: fix

**Review by mdsoftwarelab — APPROVED** [review]

**Pull request merged**